iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractTapChangerTest.java

```diff
@@ -202,6 +206,36 @@ public void testDefaultPhaseTapChangerStep() {
         assertEquals(1.0, step.getRho(), 0.0);
     }
 
+    @Test
+    public void testAdderDefaultLoadTapChangingCapabilities() {
```

Comment:
In this test, can you also create a `RatioTapChanger` without calling `setLoadTapChangingCapabilities` then assert that it has no load tap changing capabilities? (The default value in the setter is different.)